spec/system/articles/user_visits_articles_by_tag_spec.rb

```diff
@@ -38,19 +38,19 @@
 
```

Comment:
fixed flaky spec

Reply:
What flaky spec did you fix with this? Bc it is failing a lot with this addition. any title with a special character breaks https://github.com/thepracticaldev/dev.to/pull/8714